volatility/framework/plugins/windows/cachedump.py

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
These don't include the volatility copyright header.  We'll be happy to include them in the community plugins area, but we can't include them in the core framework unless you append that copyright header to the file I'm afraid.  Please ensure that all the code you're submitting is in agreement with the terms of the license.